plugins/rss/plugin.spec.yaml

```
@@ -27,6 +27,7 @@ connection:
     type: string
     description: Feed URL
     required: true
+    example: https://example.com
```

Would be good to give a more specific example of what a RSS URL looks like

```suggestion
    example: https://example.com/rss/current
```